config/routes.rb

```diff
@@ -44,7 +44,7 @@
 
   resource :skip_send_application, only: [:create]
 
-  resources :steps, only: %i[index show] do
+  resources :steps, only: %i[index show update] do
```

Comment:
do we call `update ` on `steps` ? I thought we only ever used the `update` action for the nested resources inside

Reply:
We call update on the classes that inherit from this. Never directly on this controller.